src/parsers/ddi_parser.jl

```diff
@@ -0,0 +1,174 @@
+
+
+"""
+    parse_ddi(filepath::String)
+
+Parses a valid IPUMs DDI XML file and returns a DDIInfo object containing 
+the IPUMs extract metadata. 
+
+### Arguments
+
+- `filepath::String` -- A string containing the path to the IPUMS DDI XML file. 
+
+### Returns
+
+A DDIInfo object that contains all of the file-level and variable-level 
```

Comment:
Tiny comment: could you put DDInfo in graveticks (i.e. `DDIInfo`)? I think this allows Documenter to cross-link within documentation.

Reply:
Could you also leave a note for folks to see the documentation for a `DDIInfo` object for more information?

Reply:
Resolved. This should be set now. I added the backticks for `DDIInfo`. I also included a note to look at the `DDIInfo` docs for more info.